src/policy/largeobjectspace.rs

```diff
@@ -153,7 +153,6 @@ impl<VM: VMBinding> LargeObjectSpace<VM> {
 
     pub fn prepare(&mut self, full_heap: bool) {
         if full_heap {
-            debug_assert!(self.treadmill.from_space_empty());
             self.mark_state = MARK_BIT - self.mark_state;
         }
         self.treadmill.flip(full_heap);
```

Comment:
I feel that flip the from/to space will leak dead objects. After the first trace, live objects have been copied to the to space and what left in from space are dead objects. Now after the flip, the from space will become the to space, and those dead objects will be treated as live. At this stage, sweep has not been executed yet.

Reply:
You are right. I will work on this.